logstash-core/src/main/java/org/logstash/common/io/DeadLetterQueueWriter.java

```diff
@@ -173,7 +173,7 @@ public void close() {
         }
     }
 
-    static Stream<Path> getSegmentPaths(Path path) throws IOException {
+    static Stream<Path> listSegmentPaths(Path path) throws IOException {
```

Comment:
WDYT about moving this to `DeadLetterQueueUtils` as it is shared by both reader and writer?

Reply:
Fixed by 6fc26a792753c0f74c0e82ebccc2e312dbbea852